Documentation/automatic-bootstrap-artifact.md

```diff
@@ -0,0 +1,70 @@
+Automatic bootstrap Artifact
+============================
+
+The bootstrap Artifact provides a way to populate the Mender client database when it is first
+initialized, since it is difficult to update the binary database from the build. Among other things,
+this enables delta updates to be performed using the very first update, instead of needing to do one
+rootfs update first to populate the `artifact_provides` attributes.
+
+Properties of the bootstrap Artifact
+------------------------------------
+
+A bootstrap Artifact is an empty Artifact which provides only meta-data. Specifically, it contains
+`artifact_provides` key/value pairs.
+
+To be considered a bootstrap Artifact, the Artifact must comply with the following requirements:
+* Have no `scripts`
+* Have no `artifact_depends`
+* Contain exactly 1 update which:
+  * Contains no files
```

Comment:
Maybe also mention: "Contains an empty payload, made with the `write bootstrap-artifact` argument of the mender-artifact tool."